pom.xml

```diff
@@ -44,7 +44,7 @@
     <appframeworkVersion>2.2.1</appframeworkVersion>
     <uiframeworkVersion>3.3.1</uiframeworkVersion>
     <uicommonsVersion>1.4</uicommonsVersion> 
-    <webservices.restVersion>2.12</webservices.restVersion> 
+    <webservices.restVersion>2.22.0</webservices.restVersion>
```

Comment:
Was this in order to get access to `Uploadable`? If yes I think that you can release the dependency constraint to **2.17**. Could you check?

Reply:
Yes, and checked !